pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.4.3</version>
+		<relativePath/>
+	</parent>
+	<groupId>org.folio</groupId>
+	<artifactId>edge-api-utils</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.0-SNAPSHOT</version>
+  <name>edge-api-utils</name>
+  <description>Common/Shared Edge API utilities for edge-common and edge-common-spring</description>
+
+	<properties>
+		<java.version>11</java.version>
+		<vault.version>3.1.0</vault.version>
+		<aws-java-sdk.version>1.11.313</aws-java-sdk.version>
+		<httpcore.version>4.4.9</httpcore.version>
+		<versions-maven-plugin.version>2.8.1</versions-maven-plugin.version>
+		<maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
+		<maven-source-plugin.version>3.2.1</maven-source-plugin.version>
+		<build-helper-maven-plugin.version>3.2.0</build-helper-maven-plugin.version>
+		<maven-release-plugin.version>3.0.0-M1</maven-release-plugin.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-logging</artifactId>
+    </dependency>
```

Comment:
I was hoping nothing specific to spring nor vertx would be required here

Reply:
instead of `spring-boot-starter-logging` I can use `log4j-to-slf4j` for logging all classes, like: VaultStore, SecureStoreFactory and etc. Does that make sense?

Reply:
I deleted spring from this module